packages/zapp/console/src/components/Launch/LaunchForm/LaunchFormDialog.tsx

```diff
@@ -0,0 +1,38 @@
+import { Dialog } from '@material-ui/core';
+import * as React from 'react';
+import { LaunchForm } from 'components/Launch/LaunchForm/LaunchForm';
+import { ResourceIdentifier, ResourceType } from 'models/Common/types';
+
+function getLaunchProps(id: ResourceIdentifier) {
+  if (id.resourceType === ResourceType.TASK) {
+    return { taskId: id };
+  }
+
+  return { workflowId: id };
```

Comment:
nit: in future the different resource type can be provided here, which will break things without a clear reason why.

In such cases it's better to either:

- Use switch/case statement with `default` return which includes debug.log() in case if unexpected resource type was provided

OR

- Use if statemets for all expected types, and regular return at the ens of the function for unexpected